modules/web/src/app/wizard/step/cluster/template.html

```diff
@@ -14,6 +14,13 @@
 limitations under the License.
 -->
 
+<ng-container *ngIf="loadingClusterDefaults">
+  <mat-spinner [diameter]="25"
+               class="km-spinner"
+               color="accent"></mat-spinner>
+</ng-container>
+
+
 <form [formGroup]="form">
```

Comment:
Should we hide the form when cluster defaults are loading?

Reply:
https://user-images.githubusercontent.com/17727069/208647430-f0c79291-1904-4949-94b8-402f9c73ffaa.mp4

Note for reviewer: (Technical note)
- Instead of wrapping form on ngTemplate i have just negated condtion to have smaller changset